### PR TITLE
Fix when using plugin on windows

### LINKED ISSUE #20: Is the sample broken ?
tried on windows 10. 
It seems to search file on C drive... while the repo is cloned on G: drive...
As you can read in the log

- Steps to reproduce (python 3.8.3)

pip install mkdocs-monorepo-plugin
mkdocs serve

```
G:\DXC\JET\mkdocs-monorepo-plugin\sample-docs (master -> origin)                                                    
λ mkdocs serve                                                                                                      
INFO    -  Building documentation...                                                                                
INFO    -  Cleaning site directory                                                                                  
ERROR   -  Error reading page 'versions\v1\changelog.md': 'C:\\Users\\LDELAP~1\\AppData\\Local\\Temp\\docs_f5_vyqpp\
\versions\\v1\\changelog.md'                                                                                        
Traceback (most recent call last):                                                                                  
  File "g:\applis\python383\lib\runpy.py", line 194, in _run_module_as_main                                         
    return _run_code(code, main_globals, None,                                                                      
  File "g:\applis\python383\lib\runpy.py", line 87, in _run_code                                                    
    exec(code, run_globals)                                                                                         
  File "G:\APPLIS\Python383\Scripts\mkdocs.exe\__main__.py", line 7, in <module>                                    
  File "g:\applis\python383\lib\site-packages\click\core.py", line 829, in __call__                                 
    return self.main(*args, **kwargs)                                                                               
  File "g:\applis\python383\lib\site-packages\click\core.py", line 782, in main                                     
    rv = self.invoke(ctx)                                                                                           
  File "g:\applis\python383\lib\site-packages\click\core.py", line 1259, in invoke                                  
    return _process_result(sub_ctx.command.invoke(sub_ctx))                                                         
  File "g:\applis\python383\lib\site-packages\click\core.py", line 1066, in invoke                                  
    return ctx.invoke(self.callback, **ctx.params)                                                                  
  File "g:\applis\python383\lib\site-packages\click\core.py", line 610, in invoke                                   
    return callback(*args, **kwargs)                                                                                
  File "g:\applis\python383\lib\site-packages\mkdocs\__main__.py", line 133, in serve_command                       
    serve.serve(                                                                                                    
  File "g:\applis\python383\lib\site-packages\mkdocs\commands\serve.py", line 141, in serve                         
    config = builder()                                                                                              
  File "g:\applis\python383\lib\site-packages\mkdocs\commands\serve.py", line 136, in builder                       
    build(config, live_server=live_server, dirty=dirty)                                                             
  File "g:\applis\python383\lib\site-packages\mkdocs\commands\build.py", line 271, in build                         
    _populate_page(file.page, config, files, dirty)                                                                 
  File "g:\applis\python383\lib\site-packages\mkdocs\commands\build.py", line 160, in _populate_page                
    page = config['plugins'].run_event(                                                                             
  File "g:\applis\python383\lib\site-packages\mkdocs\plugins.py", line 94, in run_event                             
    result = method(item, **kwargs)                                                                                 
  File "g:\applis\python383\lib\site-packages\mkdocs_monorepo_plugin\plugin.py", line 62, in on_pre_page            
    page.file.abs_src_path = self.files_source_dir[page.file.abs_src_path]                                          
KeyError: 'C:\\Users\\LDELAP~1\\AppData\\Local\\Temp\\docs_f5_vyqpp\\versions\\v1\\changelog.md'                    
                                                                                                                    
```

### PR DESCRIPTION
**Pull request by hooloovooo**
Fixes #20

### CODE REVIEW

File: mkdocs_monorepo_plugin/merger.py [hunk 1]
```diff
@@ -55,8 +55,8 @@ def merge(self):
                 source_dir = docs_dir
                 dest_dir = self.temp_docs_dir.name
             else:
-                source_dir = "{}/docs".format(docs_dir)
-                dest_dir = "{}/{}".format(self.temp_docs_dir.name, alias)
+                source_dir = os.path.join(docs_dir, "docs")
+                dest_dir = os.path.join(self.temp_docs_dir.name, alias.replace("/", "-"))
```
Comment (rossmechanic) [inline mkdocs_monorepo_plugin/merger.py:60]: @bih for some reason this change trigger a `A relative path to ... is included in the 'nav' configuration, which is not found in the documentation files`. Not sure what could have triggered that but it only appears in 0.4.9 and works fine in 0.4.8

Comment (rossmechanic) [reply mkdocs_monorepo_plugin/merger.py:60]: Ah I figured it out. It occurs if the site name has a `/` in it. If that case, it converts the `/` into `-`, which breaks the path

Comment (rossmechanic) [reply mkdocs_monorepo_plugin/merger.py:60]: @emmaindal

Comment (bih) [reply mkdocs_monorepo_plugin/merger.py:60]: @rossmechanic Ah interesting, thanks for letting us know. We can investigate. Could you create a GitHub issue for this?

By the way, if you want to receive more direct support from us, I would recommend joining the [Backstage discord](https://discord.gg/MUpMjP2) and hang out in the #docs-like-code channel.

**Pull request merged**